content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -290,6 +290,10 @@ kubectl logs -f my-pod                              # stream pod logs (stdout)
 kubectl logs -f my-pod -c my-container              # stream pod container logs (stdout, multi-container case)
 kubectl logs -f -l name=myLabel --all-containers    # stream all pods logs with label name=myLabel (stdout)
 kubectl run -i --tty busybox --image=busybox -- sh  # Run pod as interactive shell
+kubectl run nginx --image=nginx --restart=Never -n  # Run pod nginx in a specific namespace
```

Comment:
The below `mynamespace` should follows `-n` here.

Reply:
@xiangpengzhao  Please review my changes thanks :)

Reply:
The change LGTM.
/cc @erictune @krousey 
for another review.